src/components/spend.rs

```diff
@@ -12,6 +12,7 @@ use crate::{
     sign::sign_resolution_tx,
     tx::resolution_tx,
     util::{P2TR_TX_VBYTE_KEY_PATH, parse_network, parse_nsec},
+    validation::*,
```

Comment:
no globbing import the stuff explicitly, this is not Python...
We only glob preludes like `dioxus::prelude::*` or in tests with `use super::*`